core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -45,8 +45,8 @@ import org.apache.spark.util.{ShutdownHookManager, Utils}
  *
  * ShuffleDataIO also can change the behavior of deleteFilesOnStop.
  */
-private[spark] class DiskBlockManager(conf: SparkConf, var deleteFilesOnStop: Boolean)
-  extends Logging {
+private[spark] class DiskBlockManager(conf: SparkConf, var deleteFilesOnStop: Boolean,
+    isDriver: Boolean) extends Logging {
```

Comment:
Tiny style nit: wrapping long line to conform with [style guide](https://github.com/databricks/scala-style-guide#spacing-and-indentation)

```suggestion
private[spark] class DiskBlockManager(
    conf: SparkConf,
    var deleteFilesOnStop: Boolean,
    isDriver: Boolean)
  extends Logging {
```